src/intents/IntentsConfiguration.ts

```diff
@@ -0,0 +1,69 @@
+import { StandardIntent } from './Intents';
+import { StandardContextType } from '../context/ContextType';
+
+export const IntentsConfiguration = {
+  CreateInteraction: ['fdc3.contactList', 'fdc3.interaction', 'fdc3.nothing'],
+  StartCall: ['fdc3.contact', 'fdc3.contactList', 'fdc3.nothing'],
+  StartChat: ['fdc3.contact', 'fdc3.contactList', 'fdc3.chat.initSettings', 'fdc3.nothing'],
+  StartEmail: ['fdc3.email', 'fdc3.nothing'],
+  SendChatMessage: ['fdc3.chat.message', 'fdc3.nothing'],
+  ViewAnalysis: ['fdc3.instrument', 'fdc3.organization', 'fdc3.portfolio', 'fdc3.nothing'],
+  ViewChat: ['fdc3.chat.room', 'fdc3.contact', 'fdc3.contactList', 'fdc3.nothing'],
+  ViewChart: [
+    'fdc3.chart',
+    'fdc3.instrument',
+    'fdc3.instrumentList',
+    'fdc3.portfolio',
+    'fdc3.position',
+    'fdc3.nothing',
+  ],
+  ViewContact: ['fdc3.contact', 'fdc3.nothing'],
+  ViewHoldings: ['fdc3.instrument', 'fdc3.instrumentList', 'fdc3.organization', 'fdc3.nothing'],
+  ViewInstrument: ['fdc3.instrument', 'fdc3.nothing'],
+  ViewInteractions: ['fdc3.contact', 'fdc3.instrument', 'fdc3.organization', 'fdc3.nothing'],
+  ViewMessages: ['fdc3.chat.searchCriteria', 'fdc3.nothing'],
+  ViewNews: [
+    'fdc3.country',
+    'fdc3.instrument',
+    'fdc3.instrumentList',
+    'fdc3.organization',
+    'fdc3.portfolio',
+    'fdc3.nothing',
+  ],
+  ViewOrders: ['fdc3.contact', 'fdc3.instrument', 'fdc3.organization', 'fdc3.nothing'],
+  ViewProfile: ['fdc3.contact', 'fdc3.organization', 'fdc3.nothing'],
+  ViewQuote: ['fdc3.instrument', 'fdc3.nothing'],
+  ViewResearch: ['fdc3.contact', 'fdc3.instrument', 'fdc3.organization', 'fdc3.nothing'],
+};
```

Comment:
I tried to disable Prettier processing for this file via `.prettierignore` or a local ESLint file, to no avail
And TSDX seems to be in maintenance mode...